proposals/interface-types/working-notes/choices/webidl.md

```diff
@@ -0,0 +1,257 @@
+# Interface Types and WebIDL
+
+## Preamble
+This note is part of a series that explain some of the major design choices in
+the Interface Types specification.
+
+## Introduction
+
+A key motivation for Interface Types is the semantic gap between the
+expressivitiy of core webAssembly and the needs of API designers and application
```

Comment:
Typographical nit: here and elsewhere, "webAssembly" is used, but that capitalization is not standard. Recommend either using "WebAssembly" or, if something shorter is desired, "Wasm".

Reply:
I will sternly tell my fingers to use WebAssembly :)